data/events/2022-lagos.yml

```
@@ -0,0 +1,94 @@
+name: "2022-lagos" # The name of the event. Four digit year with the city name in lower-case, with no spaces.
+year: "2022" # The year of the event. Make sure it is in quotes.
+city: "Lagos" # The displayed city name of the event. Capitalize it.
+event_twitter: "devopsdayslag" # Change this to the twitter handle for your event such as devopsdayschi or devopsdaysmsp
+description: "Devopsdays is coming to Lagos!" # Edit this to suit your preferences
+ga_tracking_id: "" # If you have your own Google Analytics tracking ID, enter it here. Example: "UA-74738648-1"
+cancel: "false"
+
+# All dates are in unquoted 2020-MM-DDTHH:MM:SS+TZ:TZ, like this:
+#   variable: 2019-01-04T00:00:00+02:00
+#   variable: 2019-01-05T23:59:59+02:00
+# Note: we allow 2020-MM-DD for backward compatibility, but it can lead to unexpected behaviors (like your event disappearing from the front page during your last day)
+
+startdate: "2022-10-29T10:00:0+01:00" # The start date of your event. Leave blank if you don't have a venue reserved yet.
```

The formatting here is really picky.

```suggestion
startdate: "2022-10-29T10:00:00+01:00" # The start date of your event. Leave blank if you don't have a venue reserved yet.
```